test/common/tmpdir.js

```diff
@@ -5,17 +5,27 @@ const fs = require('fs');
 const path = require('path');
 const { pathToFileURL } = require('url');
 const { isMainThread } = require('worker_threads');
+const isUnix = process.platform === 'darwin' || process.platform === 'linux';
 
 function rmSync(pathname, useSpawn) {
   if (useSpawn) {
-    const escapedPath = pathname.replaceAll('\\', '\\\\');
-    spawnSync(
-      process.execPath,
-      [
-        '-e',
-        `require("fs").rmSync("${escapedPath}", { maxRetries: 3, recursive: true, force: true });`,
-      ],
-    );
+    if (isUnix) {
+      for (let i = 0; i < 3; i++) {
+        const { status } = spawnSync(`rm -rf ${pathname}`);
```

Comment:
`pathname` may contain some chars that needs escaping
```suggestion
        const { status } = spawnSync(...common.escapePOSIXShell`rm -rf "${pathname}"`);
```
But I don't know if we can import `common` here